.gitignore

```diff
@@ -2,4 +2,43 @@
 .pioenvs
 .piolibdeps
 .vscode
-.DS_Store
```

Comment:
Please keep `.DS_Store` here, as it's a very annoying file generated by Mac OS every time you open a directory using finder.